examples/android/gui_window/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
This should not be in git.

Reply:
Removed now. I mistakenly thought it had gone when I set it to be ignored.